gateway-release/home/conf/gateway-site.xml

```diff
@@ -88,6 +88,35 @@ limitations under the License.
         <value>60</value>
         <description>The interval (in seconds) for polling Ambari for cluster configuration changes.</description>
     </property>
+    <!-- @since 2.0.0 WebShell configs -->
+    <!-- must have websocket enabled to use webshell --> 
+    <property>
+        <name>gateway.webshell.feature.enabled</name>
+        <value>false</value>
+        <description>Enable/Disable webshell feature.</description>
+    </property>
+    <property>
+        <name>webshell.max.concurrent.sessions</name>
+        <value>3</value>
+        <description>maximum number of concurrent webshell sessions</description>
```

Comment:
These are total :( what number would you recommend? does 20 sound good?

Reply:
Good question. We should tease out what might be an upper limit rather than arbitrarily assign some number. Perhaps an inventory of resources allocated per session would be useful: threads, websockets, etc?

Reply:
Good point. So, each Webshell session will create 1 WS thread and one HTTP connection. There are reports of Knox comfortably  handling more than 20 WS threads (but I will test this out). About HTTP connections, for jetty 9 min thread count is 10 and max is 1000. I need to see what we have and also test this out. I will do some a bit more load testing on this and come up with a number.